tensorflow/tools/docker/Dockerfile.devel-gpu

```diff
@@ -1,11 +1,20 @@
-FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+FROM nvidia/cuda:9.0-base-ubuntu16.04
 
 LABEL maintainer="Craig Citro <craigcitro@google.com>"
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
         build-essential \
+        cuda-command-line-tools-9-0 \
```

Comment:
Just noticed. TF also depends on `libcupti`.
Do we need to install that separately?

Reply:
I mentioned it in my original RFE: https://github.com/tensorflow/tensorflow/issues/15284

> And there is still room for improvement in CUDA 9.1 when CUPTI has its own package (today we have to pull cuda-command-line-tools-9-0), or if we disable CUPTI tracing (not sure if possible).

In this image with need the compiler toolchain anyway, so CUPTI comes with it.

Reply:
```sh
$ dpkg -S /usr/local/cuda-9.0/extras/CUPTI/lib64/libcupti.so.9.0.176
cuda-command-line-tools-9-0: /usr/local/cuda-9.0/extras/CUPTI/lib64/libcupti.so.9.0.176
```

Reply:
Looks good to me.